### PR TITLE
Add beta and machine information

### PR DESCRIPTION
**Pull request by fabyan-alexander**
beta_overview
machine_endpoints

### CODE REVIEW

**Comment by fabyan-alexander** [comment]
[[Docusaurus] Add machine endpoints](https://trello.com/c/XQvCC4Ho/196-docusaurus-add-machine-endpoints)

File: docs/beta_machines_endpoints.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+
+This feature has the following endpoints available:
+
+Description | Endpoints
+--- | ---
+[Get all machines][3] | <span class="badge badge--success">GET</span> `/users/{leafUserId}/machines`
+[Create a machine][4] | <span class="badge badge--success">POST</span> `/users/{leafUserId}/machines`
+[Get a machine][5] | <span class="badge badge--success">GET</span> `/users/{leafUserId}/machines/{machine_id}`
+[Delete a machine][6] | <span class="badge badge--success">DELETE</span> `/users/{leafUserId}/machines/{machine_id}`
+[Update a machine][7] | <span class="badge badge--success">PATCH</span> `/users/{leafUserId}/machines/{machine_id}`
```
Comment (ricardosantacattarina) [inline docs/beta_machines_endpoints.md:39]:
```suggestion
[Get a machine][5] | <span class="badge badge--success">GET</span> `/users/{leafUserId}/machines/{machineId}`
[Delete a machine][6] | <span class="badge badge--success">DELETE</span> `/users/{leafUserId}/machines/{machineId}`
[Update a machine][7] | <span class="badge badge--success">PATCH</span> `/users/{leafUserId}/machines/{machineId}`
```

File: docs/beta_machines_endpoints.md [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+  "name": "str",
+  "provider": null,
+  "providerMachineId": null,
+  "providerOrganizationId": null
+}
+```
+
+### Get a machine
+
+&nbsp<span class="badge badge--success">GET</span>  `/api/users/{leafUserId}/machines/{machine_id}`
```
Comment (ricardosantacattarina) [inline docs/beta_machines_endpoints.md:214]:
```suggestion
&nbsp<span class="badge badge--success">GET</span>  `/api/users/{leafUserId}/machines/{machineId}`
```

File: docs/beta_machines_endpoints.md [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+{ label: 'Python', value: 'py', },
+{ label: 'JavaScript', value: 'js', },
+]
+}>
+<TabItem value="js">
+
+  ```js
+  const axios = require('axios')
+  const TOKEN = 'YOUR_TOKEN'
+  const endpoint ='https://api.withleaf.io/services/beta/api/users/{leafUserId}/machines/{machineId}'
```
Comment (ricardosantacattarina) [inline docs/beta_machines_endpoints.md:232]: machineId? yeah, I prefer it over machine_id

File: docs/beta_machines_endpoints.md [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+  "name": "str",
+  "provider": null,
+  "providerMachineId": null,
+  "providerOrganizationId": null
+}
+```
+
+### Delete a machine
+
+&nbsp<span class="badge badge--success">DELETE</span>  `/api/users/{leafUserId}/machines/{machine_id}`
```
Comment (ricardosantacattarina) [inline docs/beta_machines_endpoints.md:282]:
```suggestion
&nbsp<span class="badge badge--success">DELETE</span>  `/api/users/{leafUserId}/machines/{machineId}`
```

File: docs/beta_machines_endpoints.md [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+
+Delete a machine data
+
+:::info Only machines created by the user can be Deleted
+Machine data obtained from providers cannot be deleted
```
Comment (ricardosantacattarina) [inline docs/beta_machines_endpoints.md:287]: Nice!

File: docs/beta_machines_endpoints.md [hunk 6]
```diff
@@ -0,0 +1,403 @@
...
+  curl -X DELETE \
+      -H 'Authorization: Bearer YOUR_TOKEN' \
+      'https://api.withleaf.io/services/beta/api/users/{leafUserId}/machines/{machineId}'
+  ```
+  </TabItem>
+</Tabs>
+
+### Update a machine
+
+&nbsp<span class="badge badge--success">PATCH</span>  `/api/users/{leafUserId}/machines/{machine_id}`
```
Comment (ricardosantacattarina) [inline docs/beta_machines_endpoints.md:338]:
```suggestion
&nbsp<span class="badge badge--success">PATCH</span>  `/api/users/{leafUserId}/machines/{machineId}`
```

**Review by ricardosantacattarina — APPROVED** [review]
Great work! I think we should stick to machineId instead of machine_id, what do you think?

File: docs/beta_machines_endpoints.md (hunk 1 above)
Comment (fabyan-alexander) [reply docs/beta_machines_endpoints.md:39]: Bull's eye!

File: docs/beta_machines_endpoints.md (hunk 4 above)
Comment (fabyan-alexander) [reply docs/beta_machines_endpoints.md:282]: Bull's eye!

File: docs/beta_machines_endpoints.md (hunk 6 above)
Comment (fabyan-alexander) [reply docs/beta_machines_endpoints.md:338]: Bull's eye!

**Review by Vinniekun — APPROVED** [review]

File: docs/beta_machines_endpoints.md (hunk 3 above)
Comment (fabyan-alexander) [reply docs/beta_machines_endpoints.md:232]: Thanks for pointing that!
Changed in https://github.com/Leaf-Agriculture/docs/pull/161/commits/95b0cf5b7ddf2dab386a912373889620c950cbae

**Pull request merged**